gpu_ocean/SWESimulators/CDKLM16.py

```diff
@@ -275,6 +300,52 @@ def __init__(self, \
         self.angle_texref.set_address_mode(1, cuda.address_mode.CLAMP)
         self.angle_texref.set_flags(cuda.TRSF_NORMALIZED_COORDINATES) #Use [0, 1] indexing
         
+        
+        
+        # Texture for coriolis f
+        self.coriolis_texref = self.kernel.get_texref("coriolis_f_tex")
+        
+        # Create the CPU coriolis
+        if (latitude is not None):
+            if (self.f != 0.0):
+                raise RuntimeError("Cannot specify both latitude and f. Make your mind up.")
```

Comment:
Tror vi bør sette default-verdi for f til `f = 0`.
Dersom vi nå ønsker å opprette en CDKLM sim med latitude array, så må vi manuelt sette f = 0, og det føles veldig feil.

Reply:
Ikke mulig å sette default f uten å ødelegge parameter-rekkefølge, og dermed brekke masse kode. Av de to onder er det minst ille å gjøre slik som inplementert mener jeg. Vi har måttet sette f=0 på alle tidligere simuleringer. Hvis du nå setter f og lat får du feilmelding, som er fornuftig.